Formula/csound.rb

```diff
@@ -0,0 +1,66 @@
+class Csound < Formula
+  desc "Sound and music computing system"
+  homepage "https://csound.com"
+  url "https://github.com/csound/csound/archive/6.12.2.tar.gz"
+  sha256 "39f4872b896eb1cbbf596fcacc0f2122fd3e5ebbb5cec14a81b4207d6b8630ff"
+
+  depends_on "cmake" => :build
+
+  depends_on "fltk"
+  depends_on "liblo"
+  depends_on "libsndfile"
+  depends_on "stk"
+
+  def install
+    inreplace "CMakeLists.txt",
+      %r{^set\(CS_FRAMEWORK_DEST\s+"~/Library/Frameworks"\)$},
+      "set(CS_FRAMEWORK_DEST \"#{frameworks}\")"
```

Comment:
Can't this be fixed by a `cmake` arg, rather than fiddling with the `CMakeLists.txt` file?

Reply:
As far as I can tell, no, unfortunately. This seems to be a hard-coded variable, not a CMake option:

https://github.com/csound/csound/blob/8722a6144e9e4d6b9fab264bc06c3741d028b427/CMakeLists.txt#L28